README.md

```diff
@@ -1 +1,49 @@
-# sldt-semantic-models
+# Aspect Models for Eclipse Tractus-X Semantic Layer (SLDT)
+The repository contains the aspect models based on [BAMM](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/snapshot/index.html) for the Tractus-X project in Catena-X.
+
+
+# Using the models
+The models can locally be processed with the [BAMM CLI](https://openmanufacturingplatform.github.io/sds-documentation/sds-documentation/index.html).
+It allows you to generate different artifacts (diagrams, example payload, java class files) out of it.
+
+To generate the commonly used set of artifacts, this repository also contains a script, `generate.sh`, which automates some steps, and might be useful for your task.
+
+Example call for the Material.ttl aspect
+```
+./generate.sh io.catenax.batch/1.0.2/Batch.ttl
+
+```
+You will find a folder Material-artifacts in the root folder of the repository, which contains the generated artifacts.
+
+Limitations of the script: You need an internet connection to allow the script to download the BAMM-CLI and the Catena-X style for the documentation. 
+Currently the script uses version 1.0.2 of the BAMM-CLI, which doesn't work properly on the Mac (graphics will not generate images [issue](https://github.com/OpenManufacturingPlatform/sds-sdk/issues/38)). 
+
+# Status of model
+Each model has its life cycle and can thus have a different status. This information always corresponds to a specific version of the model. To indicate the state of the model version in Git, one creates a file with the name `metadata.json` and places it in the same directory as the corresponding model file. An example `metadata.json` looks like this:
+
+```
+{ "status" : "deprecate"} 
+```
+
+The following table lists the possible values for `status` and what they mean:
+
+status | status in Semantic Hub | description
+----| ---- | ---- |
+draft | DRAFT | This version of the model is under development and can change at any time.
+release | RELEASED | The version of the model is considered as stable and any modifications to the model trigger a new release and subsequentially a new version. 
+deprecate | DEPRECATED | The version of the model has reached its end-of-life and should not be used anymore because it will be deleted later. 
+
+The `metadata.json` is only relevant for model files on the branch `main`. All other branches are development branches, and the models from these branches are therefore implicitly in status "DRAFT". 
```

Comment:
this will be the same also if we work with forks? I assume so but just wanted to mention

Reply:
Yes, and I would keep the hint to other branches since at least technical it is possible to have other branches in the Eclipse repository as well.